flink-connector-cassandra/archunit-violations/dc1ba6f4-3d84-498c-a085-e02ba5936201

```diff
@@ -1,6 +1,12 @@
-org.apache.flink.streaming.connectors.cassandra.CassandraConnectorITCase does not satisfy: only one of the following predicates match:\
+org.apache.flink.connector.cassandra.source.CassandraSourceITCase does not satisfy: only one of the following predicates match:\
```

Comment:
FLINK-31804 will resolve this violation.

Reply:
> [FLINK-31804](https://issues.apache.org/jira/browse/FLINK-31804) will resolve this "violation".

Once the other PR is merged and the cassandra connector uses the new rule, only `CassandraSourceITCase` "violation" will be removed. `CassandraConnectorITCase` "violation" will remain as this class does not use MiniCluster.